### ISSUE TITLE
List is not announced as list with VoiceOver

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#268]
VoiceOver doesn't announce that this is a list (highlighted in screenshot). It should say "list start" and "list end". VoiceOver should also say "bullet" for each list item but instead it says "unpronounceable, black circle" for each bullet.

iOS 13.5.1 on iPhone 7

![7](https://user-images.githubusercontent.com/14280062/85959661-af2fb500-b96b-11ea-9a6d-a765559dfd03.jpg)

**Comment by JuliannaR** [comment]
Crossed with https://github.com/cds-snc/covid-shield-mobile/issues/258

**Comment by JuliannaR** [comment]
Crossing these issues. All lists should be coded as a list. I'm combining this into one issue.
https://github.com/cds-snc/covid-shield-mobile/issues/506
https://github.com/cds-snc/covid-shield-mobile/issues/486

**Comment by JuliannaR** [comment]
https://github.com/cds-snc/covid-shield-mobile/issues/258

**Comment by qanitah** [comment]
List on "How to self-isolate" is not read as a list. 

iOS 13.5.1 on iPhone 7
Android 8.1.0 on Samsung Galaxy

![image](https://user-images.githubusercontent.com/14280062/87042420-a8057400-c1c1-11ea-9c19-792b145fa9ec.png)

**Comment by JuliannaR** [comment]
@jelen-HC have you had a chance to look at this one?

**Comment by sarasuri** [comment]
Samsung Galaxy J3 Prime
Android version 7.0

On the "No reporting yet in your area" screen, similar behaviour that Talkback reads black circle and the text for the highlighted content in yellow not as a list.

**Comment by jelen-HC** [comment]
@JuliannaR there are major issues with how the content is structured in the app for the screen reader to read them off as lists. Additionally React Native does not have an accessibilityRole for lists or listItems from what I have researched, so trying to do this with plain text poses some major issues to provide this functionality.

**Comment by JuliannaR** [comment]
This content reads with two finger gesture, and left and right swipe. @qanitah and @sarasuri I'm wondering if there is a blocker for accessibility that they are not identified as lists in app.

It works well with spoken screen content. They are short and are visually presented as a list. Thoughts?

**Comment by jelen-HC** [comment]
@JuliannaR I have the issue pretty much resolved at this point, I have basically just had to build this functionality in from scratch. The last section that I really need to finish up is the privacy policy - it needs to be converted to json as it will not work in its current form.

I will also need the French translation for:
- List Start
- List End
- Bullet

As I am not sure what this should actually be with me not knowing the language and simple English to French translators don't always seem to do the best job.

**Comment by JuliannaR** [comment]
@jelen-HC thanks for the update. I've asked @EliseKa to confirm the translation for these.

**Comment by JuliannaR** [comment]
@jelen-HC, Elise confirmed that it should be the following:

- Début de liste
- Fin de liste
- Puce

**Comment by JuliannaR** [comment]
This is fixed in the latest build.

**Comment by qanitah** [comment]
Privacy policy page is still not fixed and it has 10 different lists (build 58). #486

Get a one-time key is also not read as an ordered list (build 58). #506

**Comment by JuliannaR** [comment]
@qanitah which version of the app are you using? This was rolled out as an accessible component. Lets connect this morning to discuss.
